pkg/apis/cluster/types.go

```diff
@@ -104,6 +105,65 @@ type ClusterSpec struct {
 	// any resource that does not tolerate the Taint.
 	// +optional
 	Taints []corev1.Taint
+
+	// The model list of resource modeling in this cluster. Each modeling name and quota can be customized by the user.
```

Comment:
Comments should be started with `ResourceModels xxx`

Reply:
> Each modeling name and quota can be customized by the user.

Does the modeling `name` can be customized? I guess the name should be allowed `enumerated values`. 
Otherwise, for the name `my-customized-resource`, how do we know what it is?

Reply:
You are right. What resource names are needed in the current production environment? In this version, I only support 4 kinds of resources. If there needs more resource name, I will customize an enum structure to support more resource names.

```go
const (
	// CPU, in cores. (500m = .5 cores)
	ResourceCPU ResourceName = "cpu"
	// Memory, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
	ResourceMemory ResourceName = "memory"
	// Volume size, in bytes (e,g. 5Gi = 5GiB = 5 * 1024 * 1024 * 1024)
	ResourceStorage ResourceName = "storage"
	// Local ephemeral storage, in bytes. (500Gi = 500GiB = 500 * 1024 * 1024 * 1024)
	// The resource name for ResourceEphemeralStorage is alpha and it can change across releases.
	ResourceEphemeralStorage ResourceName = "ephemeral-storage"
)
```

Reply:
the extension name is defined by user, diffierent user may has different custom names, we can not predict the users' extension name